### PR TITLE
Add Stookalert integration

### PR DESCRIPTION
**Pull request by fwestenberg**
**Description:**
A new integration for the Dutch RIVM Stookalert.

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/stookalert.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Stookalert"
+description: "Instructions on how to use Stookalert data within Home Assistant"
+logo: stookalert.png
+ha_category:
+  - binary-sensor
+  - environment
+ha_release: 0.103
```
Comment (fabaff) [inline source/_integrations/stookalert.markdown]: Needs to point to the release in which the integration was released.

File: source/_integrations/stookalert.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ha_iot_class: Cloud Polling
+---
+
+The `stookalert` sensor platform queries the [RIVM stookalert](https://www.rivm.nl/stookalert) API for unfavorable weather conditions or poor air quality. With a Stookalert, the RIVM calls on people not to burn wood. This can prevent health problems in people in the area.
+
+</div>
```
Comment (fabaff) [inline source/_integrations/stookalert.markdown]: Remove this.

File: source/_integrations/stookalert.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - platform: stookalert
+    province: YOUR_NL_PROVINCE
+```
+
+{% configuration %}
+province:
+  description: a province to use for the binary sensor's state. Currently only provinces in the Netherlands are supported. Valid values are `drenthe`, `flevoland`, `friesland`, `gelderland`, `groningen`, `limburg`, `noord-brabant`, `noord-holland`, `overijssel`, `utrecht`, `zeeland` or `zuid-holland`.
+  required: true
+  type: string
+scan_interval:
```
Comment (fabaff) [inline source/_integrations/stookalert.markdown]: Remove this `scan_interval` is covered at the entity level.

File: source/_integrations/stookalert.markdown (hunk 1 above)
Comment (fwestenberg) [reply source/_integrations/stookalert.markdown]: Thanks for your comments! 
What release would this be?

File: source/_integrations/stookalert.markdown (hunk 2 above)
Comment (fwestenberg) [reply source/_integrations/stookalert.markdown]: Please explain why this should be removed. In almost all integrations a line like this exists. For instance: https://www.home-assistant.io/integrations/weather.darksky/

File: source/_integrations/stookalert.markdown [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Stookalert"
+description: "Instructions on how to use Stookalert data within Home Assistant"
+logo: stookalert.png
+ha_category:
+  - binary-sensor
```
Comment (klaasnicolaas) [inline source/_integrations/stookalert.markdown]:
```suggestion
  - Binary Sensor
  - Environment
```

File: source/_integrations/stookalert.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_integrations/stookalert.markdown]: He means that you must remove the `div` tag

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
63Kb for the simple PNG provided feels a bit overkill.
Could you reduce the file size by optimizing the image? Thanks!

**Comment by fwestenberg** [comment]
> 63Kb for the simple PNG provided feels a bit overkill.
> Could you reduce the file size by optimizing the image? Thanks!

Done

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request merged**